### ISSUE TITLE
Not available on UK store...

### ISSUE THREAD

**Issue by ConfusedVorlon** [zoejessica/AccessControlKitty#3]
Hi,
trying to open this on the UK store, I get the error 'not available on this store'
Is there a reason to limit this - or is this an oversight?

thanks.

**Comment by mrousavy** [comment]
Would make a lot more sense to provide this extension via **Homebrew**, since it's also not available in my Store (Austria). 

Also it's a lot more convenient to use Homebrew than the App Store, especially for things like development plugins!

![Screenshot 2020-08-10 at 10 29 53](https://user-images.githubusercontent.com/15199031/89764343-8109cd00-daf4-11ea-8322-e3d9f7c62835.png)

**Comment by ConfusedVorlon** [comment]
I dunno - I like the simplicity & auto-updating of app store.
I know I can manage that through brew - but I tend not to.

**Comment by paulustwo** [comment]
Yep, no time to mess with this. Not available on App Store.  Move on.
